content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,856 @@
+|Type|Full|
+|Tags|`spectro-cloud-education, app:hello-universe, terraform_managed:false`|
+
+Click on **Next** to continue. 
+<br />
+
+
+### Cloud Type
+In the **Cloud Type** section, choose **Edge Native** as the infrastructure provider for this tutorial, and click on **Next** at the bottom to move on to the next section.   
+<br />
+
+### Profile Layers
+In the **Profile Layers** section, first add the [BYOS Edge OS](/integrations/byoos) pack to the OS layer
+
+|**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+|---|---|---|---|
+|OS|Public Repo|BYOS Edge OS|`1.0.0`|
+
+Replace the OS layer manifest with the following custom manifest so that the cluster profile can pull the provider image from the *ttl.sh* image registry. Recall that CanvOS returned the following custom manifest after building the Edge artifacts. The `system.xxxxx` attribute values in the manifest below are as same as what you defined in the **.arg** file while building the Edge artifacts. You must verify and change these attributes' values, as applicable to you, before using them in your cluster profile.     
+<br />
+
+```yaml
+pack:
+  content:
+    images:
+      - image: "{{.spectro.pack.edge-native-byoi.options.system.uri}}"
+options:
+  system.uri: "{{ .spectro.pack.edge-native-byoi.options.system.registry }}/{{ .spectro.pack.edge-native-byoi.options.system.repo }}:{{ .spectro.pack.edge-native-byoi.options.system.k8sDistribution }}-{{ .spectro.system.kubernetes.version }}-{{ .spectro.pack.edge-native-byoi.options.system.peVersion }}-{{ .spectro.pack.edge-native-byoi.options.system.customTag }}"
+  system.registry: ttl.sh
+  system.repo: ubuntu
+  system.k8sDistribution: k3s
+  system.osName: ubuntu
+  system.peVersion: v3.4.3
+  system.customTag: demo
+  system.osVersion: 22
+``` 
+<br />
+
+ 
+The screenshot below shows you how to reference your provider OS image in a cluster profile by using the utility build output with the BYOOS pack. 
+<br />
+
+![A screenshot of k3s OS layer in a cluster profile.](/tutorials/edge-native/clusters_edge_deploy-cluster_edit-profile.png)
+
+
+<WarningBox>
+
+As a second reminder, *ttl.sh* is a short-lived image registry. If you do not use the provider image in your cluster profile within 24 hours of pushing to *ttl.sh*, they will no longer exist and must be re-pushed.
+
+</WarningBox>
+<br />
+
+Click on the **Next layer** button to add the following Kubernetes layer to your cluster profile. 
+
+
+|**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+|---|---|---|---|
+|Kubernetes|Public Repo|Palette Optimized K3s|`1.25.x`|
+
+
+Choose the K3s version 1.25.x because you pushed the provider image compatible with K3s v1.25.2 to the *ttl.sh* image registry earlier in this tutorial. BYOOS pack's `system.uri` attribute will reference the Kubernetes version you select using the `{{ .spectro.system.kubernetes.version }}` [macro](/clusters/cluster-management/macros).
```

Comment:
```suggestion
Select the K3s version 1.25.x. 1.25.X is used because you pushed a provider image compatible with K3s v1.25.2 to the *ttl.sh* image registry earlier in this tutorial. BYOOS pack's `system.uri` attribute will reference the Kubernetes version you select using the `{{ .spectro.system.kubernetes.version }}` [macro](/clusters/cluster-management/macros).
```